contrib/release/submit-release.sh

```diff
@@ -51,12 +50,21 @@ if $GENERATE_SUMMARY; then
     CHANGELOG=$SUMMARY
     SUMMARY="$RELEASE-pr-$(date --rfc-3339=date).txt"
     echo "Prepare for release $RELEASE" > $SUMMARY
-    tail -n+4 $CHANGELOG >> $SUMMARY
+    if [ "$BRANCH" = "master" ]; then
+        echo "" >> $SUMMARY
+        echo "See the included CHANGELOG.md for a full list of changes." >> $SUMMARY
```

Comment:
FWIW I set this like this because RCs almost always have a changelog too big for the GH issue template. It'd be nice to have a more robust way to detect this for the .0 release as well since we'll include full changelog there as well I think, but :shrug: this is good enough for now.